### PR TITLE
PSR-127

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Add digital jar open task to tableview

### CODE REVIEW

File: Psorcast/PsorcastValidation/DigitalJarOpen.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "taskIdentifier"            : "DigitalJarOpen",
+    "schemaIdentifier"          : "DigitalJarOpen",
+    "progressMarkers":[],
+    "steps":
+    [
+        {
+            "identifier": "introduction",
+            "type":"overview",
+            "title":"Digital Jar Open",
+            "text":"In this activity you will place your phone on a flat surface, and rotate it several differnt distances and directions.",
```
Comment (DanWebster) [inline Psorcast/PsorcastValidation/DigitalJarOpen.json]:
```suggestion
            "text":"In this activity, you will place your phone on a flat surface and rotate it like you are opening a jar in order to measure joint function.",
```

File: Psorcast/PsorcastValidation/DigitalJarOpen.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            "image": {
+                "type":"fetchable",
+                "imageName":"PsoriasisAreaPhoto3",
+                "placementType":"topBackground"
+            },
+            "icons": [
+                {
+                    "icon":"FlatFloorIcon",
+                    "title": "A FLAT FLOOR"
```
Comment (DanWebster) [inline Psorcast/PsorcastValidation/DigitalJarOpen.json]:
```suggestion
                    "title": "A FLAT SURFACE"
```

File: Psorcast/PsorcastValidation/DigitalJarOpen.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            "text":"Let’s first begin by placing your phone on a flat surface, like a table, with the bottom of the phone facing towards you.",
+            "image":{
+                "type":"fetchable",
+                "imageName": "Foot1",
+                "placementType":"topBackground"
+            },
+            "actions":{
+                "goForward":{
+                    "type":"default",
+                    "buttonTitle":"The phone is on a flat surface"
```
Comment (DanWebster) [inline Psorcast/PsorcastValidation/DigitalJarOpen.json]:
```suggestion
                    "buttonTitle":"Phone is on a flat surface"
```

**Comment by dephillipsmichael** [comment]
> Looks great and I know this is not the final product, but I can't help myself from making ticky-tacky edits in the copy (made as suggestions on this PR, so you should just be able to make the changes inline on Github)

You will have to show me how to do the inline changes, really cool.  I merged them in.

**Pull request merged**